app/design/frontend/Magento/elogic-theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -0,0 +1,164 @@
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common =true) {
+
+    .cms-home.cms-index-index {
+        .page-wrapper > main.page-main {
+            max-width: 1920px;
+            padding-left: 0;
+            padding-right: 0;
+        }
+        .nav-sections {
+            margin-bottom: 0;
+        }
+    }
+
+
+    body {
+        .lib-list-reset-styles(@_margin: 0, @_padding: 0) {
+        .lib-css(margin, @_margin);
+        .lib-css(padding, @_padding);
+        list-style: none none;
+    }
+        .lib-css(color, @color-elogic-gray-main);
+        .lib-css(font-family, @font-family-name__base);
+    }
+
+    h1 {
+        .lib-css(font-weight, @font-weight-semibold);
+        .lib-css(color, @color-elogic-black);
+        text-transform: uppercase;
+        letter-spacing: 0.15rem;
+        transition: all 0.2s ease;
+        -webkit-transition: all 0.2s ease;
+        -moz-transition: all 0.2s ease;
+        -ms-transition: all 0.2s ease;
+        -o-transition: all 0.2s ease;
+
+        span {
+            .lib-css(font-weight, @font-weight-light);
+        }
+    }
+
+    .banner-text {
+        h1 {
+            .lib-css(font-size, @font-size__xl_xl);
+            margin-bottom:10px;
+        }
+    }
+
+    a {
+        .lib-css(color, @color-elogic-black);
+        transition: all 0.2s ease;
+        -webkit-transition: all 0.2s ease;
+        -moz-transition: all 0.2s ease;
+        -ms-transition: all 0.2s ease;
+        -o-transition: all 0.2s ease;
+    }
+
+    .banner-button {
+        a {
+            &:hover {
+                .lib-css(color, @color-elogic-gray-main);
+                border-bottom: 1px solid @color-elogic-gray-main;
+            }
+
+            &:active {
+                .lib-css(color, @color-elogic-gray-main);
+                border-bottom: 1px solid @color-elogic-gray-main;
+            }
+            text-transform: uppercase;
+            text-decoration: none;
+            border-bottom: 1px solid @color-elogic-black;
+            padding-bottom: 0.5rem;
+        }
+
+        padding-top: 5rem;
+    }
+
+    .banner_container {
+        background-color: @color-elogic-gray-banner;
+        display: grid;
+        height: auto;
+        align-items: center;
```

Comment:
Please check utilities mixins and use it for flex css rules:
code/vendor/magento/magento2-base/lib/web/css/source/lib/_utilities.less